post_installation_configuration/aws-compute-edge-zone-tasks.adoc

```diff
@@ -0,0 +1,81 @@
+
+// Edge compute pools and AWS edge zones
+include::modules/edge-machine-pools-aws-local-zones.adoc[leveloffset=+2]
+
+[id="post-install-extend-existing-to-local-zones-mtu"]
+=== Changing the Cluster Network MTU to support AWS edge zone subnets
+
+You might need to change the maximum transmission unit (MTU) value for the cluster network so that your cluster infrastructure can support {zone-type} subnets.
+
+// About the cluster MTU
+include::modules/nw-cluster-mtu-change-about.adoc[leveloffset=+3]
+
+// Changing the cluster network MTU
+include::modules/nw-cluster-mtu-change.adoc[leveloffset=+3]
+
+// Opting in to AWS Local Zones or Wavelength Zones
+include::modules/installation-aws-add-zone-locations.adoc[leveloffset=+2]
+
+// Create network requirements in an existing VPC that uses AWS Local Zones or Wavelength Zones
+include::modules/post-install-extend-existing-to-zone-type.adoc[leveloffset=+2]
+
+// Creating a VPC carrier gateway
+include::modules/installation-creating-aws-vpc-carrier-gw.adoc[leveloffset=+2]
+
+// CloudFormation template for the VPC Carrier Gateway
+include::modules/installation-cloudformation-vpc-carrier-gw.adoc[leveloffset=+2]
+
+// Creating a subnet in an AWS Local Zones or Wavelength Zones
+include::modules/installation-creating-aws-vpc-subnets-edge.adoc[leveloffset=+2]
+
+// CloudFormation template for the VPC Subnet
+include::modules/installation-cloudformation-subnet-localzone.adoc[leveloffset=+2]
+
+// Creating a machine set manifest for an AWS Local Zones or Wavelength Zones node
+include::modules/post-install-edge-aws-extend-machineset.adoc[leveloffset=+2]
+
+// Sample YAML for a compute machine set custom resource on AWS
+include::modules/machineset-yaml-aws.adoc[leveloffset=+3]
+
+// Creating a compute machine set
+include::modules/machineset-creating.adoc[leveloffset=+3]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../installing/installing_aws/installing-aws-localzone.adoc#installing-aws-localzone[Installing a cluster on AWS with compute nodes on AWS Local Zones]
+* xref:../installing/installing_aws/installing-aws-wavelength-zone.adoc#installing-aws-wavelength-zone[Installing a cluster on AWS with compute nodes on AWS Wavelength Zones]
+
+// Creating user workloads in AWS Local Zones or Wavelength Zones
+include::modules/installation-extend-edge-nodes-aws-local-zones.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../installing/installing_aws/installing-aws-localzone.adoc#installing-aws-localzone[Installing a cluster on AWS with compute nodes on AWS Local Zones]
+* xref:../installing/installing_aws/installing-aws-wavelength-zone.adoc#installing-aws-wavelength-zone[Installing a cluster on AWS with compute nodes on AWS Wavelength Zones]
+* xref:../nodes/scheduling/nodes-scheduler-taints-tolerations.adoc#nodes-scheduler-taints-tolerations-about_nodes-scheduler-taints-tolerations[Understanding taints and tolerations]
+
+[id="next-steps_aws-zone-tasks"]
```

Comment:
```suggestion
[id="next-steps-aws-zone-tasks_{context}"]
```
unless you don't want the context tag

Reply:
Don't seems to be required as per the guidelines doc: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies